types/react-native/index.d.ts

```diff
@@ -3728,7 +3728,7 @@ export interface SectionListProperties<ItemT> extends ScrollViewProperties {
     /**
      * Rendered in between adjacent Items within each section.
      */
-    ItemSeparatorComponent?: React.ComponentClass<any> | (() => React.ReactElement<any>) | null
+    ItemSeparatorComponent?: React.ComponentType<any> | (() => React.ReactElement<any>) | null
```

Comment:
Please remove the second type in the union, which is a part of `ComponentType`.

Reply:
done, thanks!